providers/manage.rb

```diff
@@ -74,7 +74,7 @@ def cert_file_resource(path, content, options = {})
     group new_resource.group
     mode(options[:private] ? 00640 : 00644)
     variables :file_content => content
-    only_if { content }
+    only_if { content ? ! content.empty? : false}
```

Comment:
This also changes the behaviour. Before the block was evaluated if content was something like `""`, `[]` or `{}`. An empty string will evaluate to `true` and then the whole statement returns `false` because `"".empty?` is `true`. Is that the intended behaviour?